wiki/Beatmaps/Beatmapsets/en.md

```diff
@@ -1,10 +1,16 @@
+---
+tags:
+  - beatmap
+  - mapset
+---
+
 # Beatmapsets
 
 <!-- TODO: connect to the rest of the wiki, ultimately this should replace the Glossary entry and be clearly distinguished from "beatmaps" -->
 
-![An in-game screenshot of a beatmapset in the song selection menu](img/in-game.jpg "Sing's beatmapset of \"nanobii - popsicle beach\"")
+![An in-game screenshot of a beatmapset in the song selection menu](img/in-game.jpg "Beatmapset of \"nanobii - popsicle beach\" hosted by user \"Sing\"")
 
-A **beatmapset** is a set of [beatmaps](/wiki/Beatmaps) for the same song. The [beatmap listing](https://osu.ppy.sh/beatmapsets) shows all of the beatmapsets submitted online.
+A **beatmapset** is a set of [beatmaps](/wiki/Beatmaps) for the same song managed by a [beatmapset host](/wiki/Beatmaps/Beatmapsets/Beatmapset_host). The [beatmap listing](https://osu.ppy.sh/beatmapsets) shows all of the beatmapsets submitted online.
```

Comment:
too specific to the common pattern of host+guests on submitted beatmaps imo, a beatmapset has nothing to do with this by default

the hsot +guest thing could be next paragraph or something